pkg/commands/copy_test.go

```diff
@@ -451,13 +450,14 @@ func TestCopyCommand_ExecuteCommand_Extended(t *testing.T) {
 		}
 		testutil.CheckNoError(t, err)
 		// Check if "dest" dir exists with contents of srcDir
-		actual, err := ioutil.ReadDir(filepath.Join(testDir, "dest"))
+		actual, err := os.ReadDir(filepath.Join(testDir, "dest"))
 		if err != nil {
 			t.Fatal(err)
 		}
 		for i, f := range actual {
+			fi, _ := expected[i].Info()
 			testutil.CheckDeepEqual(t, expected[i].Name(), f.Name())
-			testutil.CheckDeepEqual(t, expected[i].Mode(), f.Mode())
+			testutil.CheckDeepEqual(t, fi.Mode(), fi.Mode())
```

Comment:
This change is not related to the PR goals and actually breaks the test IIUC (comparing `fi.Mode()` w/ `fi.Mode()` now).  This change should likely be undone